.github/workflows/github-ci.yml

```diff
@@ -109,7 +109,9 @@ jobs:
       run: tar --zstd -xvf libraries.tar.zstd
     - uses: ammaraskar/gcc-problem-matcher@master
     - name: Compile vlasiator (Release build)
-      run: VLASIATOR_ARCH=github_actions make -j 3
+      run: |
+        VLASIATOR_ARCH=github_actions make clean
```

Comment:
should this make clean be as its own `name` so it shows in the log? (like was added for build_tools)